addons/web/static/src/model/relational_model/relational_model.js

```diff
@@ -121,6 +122,7 @@ export class RelationalModel extends Model {
         this.config = {
             isMonoRecord: false,
             currentCompanyId: company.currentCompany.id,
```

Comment:
is this one still necessary? Would be nice to have a single key `companies` in the config (so I would also rename companiesEvalContext)